packages/blockchain-wallet-v4-frontend/src/components/Form/TextAreaDebounced/index.js

```diff
@@ -26,14 +26,17 @@ const getErrorState = (meta) => {
 class TextAreaDebounced extends React.Component {
   constructor (props) {
     super(props)
-    this.state = { value: props.input.value }
+    this.state = { value: props.input.value, init: true }
```

Comment:
`updatedValue` should be initialised in constructor, same as value I guess, there won't be need for `init`